modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/start.jsp

```diff
@@ -17,7 +17,7 @@
 <%@ include file="/data_layout_builder/init.jsp" %>
 
 <liferay-editor:resources
```

Comment:
You shouldn't need resources for `classic_editor`!

Reply:
Thanks @jbalsas. I'm having another problem with the style, apparently, the styles are not loaded in Forms, I don't know why it just appears in the Journal.

Reply:
In the case of Forms, this is due to the fact that Forms is still using AlloyEditor in some parts, apparently the ckeditor css of the `moono-lisa` style is applying a different format than `moono-lexicon`.